src/index.html

```diff
@@ -5,8 +5,354 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="logo" href="#">
+        <img
+          src="images/logo.png" alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Smartphones
+            </a>
+          </li>
+
+          <li data-qa="nav-hover" class="nav__item">
+            <a class="nav__link" href="#">
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div class="card" data-qa="card">
+        <img
+          class="card__image"
+          src="images/imac.jpeg"
+          alt="product"
+        >
+
+        <h2 class="card__text">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+        <h3 class="card__code">Product code: 195434</h3>
+
+        <figure class="card__review">
+
+          <div class="stars">
+            <div class="star_active"></div>
+            <div class="star_active"></div>
+            <div class="star_active"></div>
+            <div class="star_active"></div>
+            <div class="star"></div>
```

Comment:
```suggestion
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
```
The structure should be the same as in the previous task